specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/AcquiredPlan.json

```diff
@@ -0,0 +1,321 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2015-11-01",
+        "title": "SubscriptionsManagementClient",
+        "description": "The Admin Subscriptions Management Client."
+    },
+    "host": "adminmanagement.local.azurestack.external",
```

Comment:
>adminmanagement.local.azurestack.external [](start = 13, length = 41)

Is this really the endpoint for Stack?

Reply:
Yes. The admin endpoint for azure stack will be reachable only within the deployment and the uri could change for each deployment. adminmanagement.local.azurestack.external is the typical developer kit endpoint for stack and we are using for all our stack admin swagger files